tgui/packages/tgui/interfaces/CTFPanel.js

```diff
@@ -0,0 +1,64 @@
+import { useBackend } from '../backend';
+import { Box, Button, Section, Flex, Stack } from '../components';
+import { Window } from '../layouts';
+
+export const CTFPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const teams = data.teams || [];
+  const enabled = data.enabled || [];
+  return (
+    <Window
+      title="CTF Panel"
+      width={700}
+      height={600}>
+      <Window.Content scrollable>
+        <Section title={enabled} textAlign="center">
```

Comment:
Design wise, this section should be unwrapped, and placed above the rest (to make teams sections more distinct). Text can be placed as contents of the section to make it wrap better in smaller window sizes.